ui/desktop/electron-app/src/ipc/handlers.js

```diff
@@ -116,6 +121,21 @@ handle('toggleFullscreenWindow', () => {
  */
 handle('closeWindow', () => app.quit());
 
+/**
+ * Toggle when signout is in progress
+ * to prevent app from quitting when signout modal is visible
+ */
+handle('setSignoutInProgress', (value) => (app.signoutInProgress = value));
```

Comment:
I don't think you need this handler anymore now that you send an event from electron directly